_posts/2020-01-25-electrondisplay-v1.0.0.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2019-02-13
+title: ElectronDisplay.jl v1.0.0 Released
+categories:
+  - announcement
+author: Tony Lian
+---
+
+[ElectronDisplay.jl](https://github.com/queryverse/ElectronDisplay.jl) is a package that provides a simple UI based on [Electron](https://electronjs.org/) for displaying plots and tabular data. It works with most plotting packages and almost all data structures that represent tabular data in the Julia ecosystem. The package is a core piece of the [Queryverse](https://github.com/queryverse).
+
+## What is new?
+There are a bunch of bugfixes as well as some new features in v1.0.0. The main feature is the [plotgallery](https://github.com/julia-vscode/plotgallery), a frontend used for plot display with TypeScript and react. It support displaying multiple plots in the same window at the same time, even if those plots are of different MIME types. For example, you can display an PNG image and then plot a VegaLite plot, or multiple VegaLite plots, and they would show in a single window instead of opening a new window each time or replacing the old plots:
```

Comment:
```suggestion
There are a bunch of bugfixes as well as some new features in v1.0.0. The main new feature is the [plotgallery](https://github.com/julia-vscode/plotgallery), an interactive plot display UI that is written in TypeScript and based on React. It supports displaying multiple plots in the same window simultaniously, even if those plots are created with different plotting packages. For example, you can display a Gadfly.jl figure and then plot a VegaLite.jl plot, or multiple VegaLite.jl plots, and they would show in a single window instead of opening a new window each time or replacing the old plots.
```